### DISCUSSION TITLE
Experiences with updating "Odoo Studio for Community?"

### DISCUSSION THREAD

**Discussion by kevinberger-ch** [OCA/OpenUpgrade#4255] [Q&A] [answered]
I use the Odoo Studio for Community Module, see: https://odoo-studio.com/

Unfortunately the support is very poor, and although I have been using and developing odoo for a year, I am still new to updating. Does anyone also use this module and has experience with updating? (in my case from 15.0 to 16.0)

I am grateful for any help.

**Comment by legalsylvain** [comment]
Hi @kevinberger-ch,
Odoo studio for community has nothing to do with OCA. (odoo community association).
you should contact the company. (Trương Văn Dũng // odoo.studio.community@gmail.com noted in the website).

Kind regards.

**Reply by kevinberger-ch** [reply]
Hi @legalsylvain,
I understand that Odoo Studio for Community is not associated with the OCA (Odoo Community Association), but I am currently using OpenUpgrade for updating other components of my Odoo instance. Therefore, it seemed logical to me that I might also need OCA's OpenUpgrade tool to upgrade the Odoo Studio for Community module. Given the limited support available for this module, as I've previously mentioned, I thought this forum might be the right place to discuss my issue and possibly connect with others who are using this module alongside OCA's OpenUpgrade.

**Comment by kevinberger-ch** [comment]
Hi @legalsylvain,

I noticed that the discussion has been closed. I kindly request if it could be kept open for a little longer. This would allow me the opportunity to possibly connect with others who might have experience with the Odoo Studio for Community module and OCA's OpenUpgrade. Your consideration in this matter would be greatly appreciated, as it could be very helpful for me and perhaps others in the community facing similar challenges.

Thank you for your understanding.

**Comment by kevinberger-ch** [comment, marked as answer]
Solution: Just don't use Odoo Studio for Community module 😉
